connect_uwsgi.ini

```diff
@@ -2,23 +2,24 @@
 
 # Django-related settings
 # the base directory (full path)
-chdir           = /home/nptel/HTTP_200
+chdir           = /home/rishabh/Documents/http_200
```

Comment:
Dude, why are you pushing the path of your machine on the main server ?? Do check things before adding into or.